tasks/fix-cat2.yml

```diff
@@ -1032,7 +1032,7 @@
   tags:
       - RHEL-07-020690
 
-- name: "MEDIUM | RHEL-07-020700 | Local initialization files for local interactive users must be group-owned by the users primary group or root."
+- name: "MEDIUM | RHEL-07-020700 | PATCH | Local initialization files for local interactive users must be group-owned by the users primary group or root."
```

Comment:
I'd probably skip adding these... benchparse also doesn't include them at the block level, IIRC.

Reply:
hmm ok good to know. I did scrub through and change all the block titles to include PATCH but I can back those changes out easily if that isn't the desired end state. I thought I might need the name titles to be more consistent for doing parsing in the doc generator but really I just need the | RHEL-07-###### | identifier to be there in the second spot.

So if the consensus is no PATCH identifier in block titles then I will back those changes out of this PR.